lua/sol.lua

```diff
@@ -0,0 +1,21 @@
+return function (net_conf, gk_conf, lcore)
+
+	-- Init the Solicitor configuration structure.
+	local sol_conf = gatekeeper.c.alloc_sol_conf()
+	if sol_conf == nil then
+		error("Failed to allocate sol_conf")
+	end
+
+	sol_conf.lcore_id = lcore
+	sol_conf.pri_req_max_len = 1024
+	sol_conf.enq_burst_size = 1024
+	sol_conf.deq_burst_size = 1024
```

Comment:
I just mean that the values we are setting are too big. Set `sol_conf.enq_burst_size` and `sol_conf.deq_burst_size` with the same value of `GATEKEEPER_MAX_PKT_BURST` for now. We'll think of optimal values later.